internal/pkg/api/server/templates/http/middleware.go

```diff
@@ -70,11 +87,15 @@ func ContextMiddleware(serverDeps dependencies.ForServer, h http.Handler) http.H
 
 		// Add request ID to headers
 		w.Header().Add("X-Request-Id", requestId)
-		// Add request ID to DD span
+
+		// Update span
 		if span, ok := tracer.SpanFromContext(ctx); ok {
 			span.SetTag(ext.ResourceName, r.URL.Path)
-			span.SetBaggageItem("kac.http.request.id", requestId)
-			span.SetBaggageItem("kac.storage.host", serverDeps.StorageApiHost())
+			span.SetTag(ext.HTTPURL, r.URL.Redacted())
+			span.SetTag("http.path", r.URL.Path)
+			span.SetTag("http.query", r.URL.Query().Encode())
```

Comment:
`SetBaggageItem` by mala dane hodnoty nastavit aj do `child spans`, co aj robi.
Ale nejak chyba odoslanie tychto hodnot na server, neviem ci to este nie je dokoncene, alebo kde je problem.
Nasiel som nejake issues aj k inym jazykom, kde sa ludia na to stazuju :man_shrugging: 

Tak preto som pridal niektore dolezite tagy aj do inych spans.